ramips: update MAC address configuration for Buffalo WSR-1166DHP

ikazuhiro opened the pull request:
Convert MAC address and label_mac configuration of Buffalo WSR-1166DHP to use the generic function of OpenWrt.

This commit applies commit 770cfe9 for WCR-1166DS to WSR-1166DHP too.

ikazuhiro committed:
ramips: update MAC address configuration for Buffalo WSR-1166DHP

Convert MAC address and label_mac configuration of Buffalo
WSR-1166DHP to use the generic function of OpenWrt.

Apply commit 770cfe9 for WCR-1166DS to WSR-1166DHP too.

Tested on the device and MAC address is kept before and after this
change.

Signed-off-by: Kazuhiro Ito <kzhr@d1.dion.ne.jp>

Ansuel commented:
@ikazuhiro can you verify that the same mac is applied before and after? And add this info in the commit? Also wonder why this wasn't done in the first place. Really old target?

ikazuhiro commented:
I had confirmed that the commit had kept MAC address on the device before PR.
WSR-1166DHP was supported since 0371dabc1347fd902b1a62b050dbe56064d23567, almost 8 years ago.  And, MAC extraction was committed 975e1357e6136ab5126e8d079e80bdcde338f8a6, about 7.5 years ago.

I updated commit message and rebased the branch.
